server/pypi/packages/scipy/meta.yaml

```diff
@@ -2,11 +2,14 @@ package:
   name: scipy
   version: "1.8.1"
 
+build:
+  script_env:
+    - SCIPY_USE_PYTHRAN=0
```

Comment:
If something can be done with an environment variable, that's more maintainable than adding a patch.